### ISSUE TITLE
FEATURE: provide summary data when fetching hardware inventory for a cluster

### ISSUE THREAD

**Issue by Masber** [eth-cscs/manta#20]
At the moment manta provide a summary at the node level when asking for the hardware inventory related to a cluster:
eg
```
+---------------+-----------+---------------------------------+------------------------------------+
| Node          | 32768 MiB | AMD EPYC 7742 64-Core Processor | SS11 200Gb 2P NIC Mezz REV02 (HSN) |
+==================================================================================================+
| x1003c1s7b0n0 |  ✅ (16)  |              ✅ (2)             |               ✅ (1)               |
|---------------+-----------+---------------------------------+------------------------------------|
| x1003c1s7b0n1 |  ✅ (16)  |              ✅ (2)             |               ✅ (1)               |
|---------------+-----------+---------------------------------+------------------------------------|
| x1003c1s7b1n0 |  ✅ (16)  |              ✅ (2)             |               ✅ (1)               |
|---------------+-----------+---------------------------------+------------------------------------|
| x1003c1s7b1n1 |  ✅ (16)  |              ✅ (2)             |               ✅ (1)               |
+---------------+-----------+---------------------------------+------------------------------------+
```

It would be better if we provide a summary with the counters of each hardware component type within the cluster.
Then provide a parameters to break down per node as the example above

**Comment by Masber** [comment]
example
```
+---------------------------------+--------------+---------------------------------+------------------------------------+---------------------------------+-----------------------+---------------------------------+
| AMD EPYC 7A53 64-Core Processor | Memory (GiB) | AMD EPYC 7742 64-Core Processor | SS11 200Gb 2P NIC Mezz REV02 (HSN) | AMD EPYC 7713 64-Core Processor | NVIDIA_A100-SXM4-80GB | AMD INSTINCT MI200 (MCM) OAM LC |
+===================================================================================================================================================================================================================+
| 2                               | 5120         | 24                              | 28                                 | 2                               | 8                     | 16                              |
+---------------------------------+--------------+---------------------------------+------------------------------------+---------------------------------+-----------------------+---------------------------------+
```

**Comment by Masber** [comment]
this is implemented in v1.26.0
